pulpcore/cli/ansible/context.py

```diff
@@ -21,6 +22,11 @@ class PulpAnsibleCollectionVersionContext(PulpContentContext):
     LIST_ID = "content_ansible_collection_versions_list"
     READ_ID = "content_ansible_collection_versions_read"
     CREATE_ID = "content_ansible_collection_versions_create"
+    UPLOAD_ID = "upload_collection"
```

Comment:
Wow, this is unexpected. Is this endpoint not part of the usual pulp rest api?
Just curiosity. We can proabably not do any thing about it here.

Reply:
There is a create endpoint for collection versions, but it requires you know and supply every piece of metadata for the upload. This other endpoint is apart of the galaxy api and is much simpler to use. I think they both use the same upload task code, it's just their serializers are different.

Reply:
So this endpoint is meant to be used by the galaxy cli? In pulp_container we intentionally excluded the live/push-pull api from the openapi schema...

Reply:
```suggestion
    UPLOAD_ID: ClassVar[str] = "upload_collection"
```